### ISSUE TITLE
The plugin doesn't work on windows

### ISSUE THREAD

**Issue by myartings** [cpichard/usdassimp#2]
Just follow the tutorial , build and install the plugin , it says "Failed to open "tests/example1.ply" - Cannot determine file format for @tests/example1.ply@". I Don't have any idea about this issue.  My USD version is 23.02, built in release.  I Need help.

**Comment by cpichard** [comment]
Hi @myartings , it can happen if the PXR_PLUGINPATH_NAME environment variable is not pointing the the correct folder. 
It should be PXR_PLUGINPATH_NAME=<where_you_installed_the_plugin>/plugin/usd: adding /plugin/usd to the location where the plugin was installed. Is that environment variable set correctly ?

**Comment by myartings** [comment]
@cpichard yes, I'm sure that I have set environment variable correctly. The setting is below.

![image](https://github.com/cpichard/usdassimp/assets/4220108/0d495827-bfb2-48fe-b84d-ceeb0211164f)
![image](https://github.com/cpichard/usdassimp/assets/4220108/b1b09077-5b2e-4327-b435-ccb23b8c0f96)
![image](https://github.com/cpichard/usdassimp/assets/4220108/210b474b-23c3-47ed-aba8-87fa742dc3c0)
![image](https://github.com/cpichard/usdassimp/assets/4220108/4ca7552c-edd3-404a-bcc1-bb6abc20d4c9)
![image](https://github.com/cpichard/usdassimp/assets/4220108/b0c3cdaa-2ec1-4101-b6db-993d47504375)

**Comment by cpichard** [comment]
Ok ! thanks for checking. I just compiled it on windows as well and I have the same issue as you.

**Comment by myartings** [comment]
I set TF_DEBUG environment variable to "*" prior to launching usdcat. Here is the log.
[assimp_log.txt](https://github.com/cpichard/usdassimp/files/11865173/assimp_log.txt)

**Comment by cpichard** [comment]
Thanks for the log @myartings. This is puzzling ... , it looks like the plugin is correctly registered and loaded but usd doesn't open the ply file for some reason. I have the same issue on my machine, but will be able to debug only next week-end. I'll keep you posted

**Comment by cpichard** [comment]
Hi @myartings , It looks like the code creating the plugin factory wasn't compiled. Visual Studio removed it as it didn't have internal linkage. I pushed a fix, let me know if that works for you.
